### ISSUE TITLE
Different result from `npm test` locally and on Travis

### ISSUE THREAD

**Issue by hkdobrev** [sindresorhus/refined-github#125]
When running `npm test` locally on the latest master I get this error:

``` bash
$ npm test

> @ test ~/projects/refined-github
> xo


~/projects/refined-github/extension/diffheader.js
  42:3  error  Expected no return value  consistent-return

✖ 1 problem (1 error, 0 warnings)
npm ERR! Test failed.  See above for more details.
```

Changing `diffheader.js:37` to `return false;` of course fixes it, but I'm curious why Travis CI doesn't complain about that and my local `npm test` does.

**Comment by DrewML** [comment]
Interesting - I don't know yet what's causing that, but I'm not seeing the problem locally on latest master ATM:

```shell
┌─[andrew] [LOLMachine] [master → origin ✓] [~/Documents/refined-github]
└─▪ npm test

> @ test /Users/andrew/Documents/refined-github
> xo

┌─[andrew] [LOLMachine] [master → origin ✓] [~/Documents/refined-github]
```

**Comment by hkdobrev** [comment]
Well for reference:

```
$ npm --version
3.7.3

$ node --version
v5.9.1
```

I've run `npm install` before `npm test`.

**Comment by DrewML** [comment]
Interesting. Same version #'s over here. From what I can tell, it doesn't look like XO has that rule set with ESLint.

```shell
┌─[andrew] [LOLMachine] [master ✓] [~/Documents/xo]
└─▪ grep -ir 'consistent-return' .
./node_modules/eslint/CHANGELOG.md:* 13c7c19 Update: Exclude ES5 constructors from consistent-return (fixes #5379) (Kevin Locke)
./node_modules/eslint/CHANGELOG.md:* Fix: `consistent-return` checks the last (refs #3530, fixes #3373) (Toru Nagashima)
./node_modules/eslint/CHANGELOG.md:* Fix: Allow globalReturn in consistent-return (fixes #1868) (Brandon Mills)
./node_modules/eslint/CHANGELOG.md:* Docs: fixed typo in consistent-return docs (Vitor Balocco)
./node_modules/eslint/CHANGELOG.md:* New Rule: consistent-return (fixes #481) (Nicholas C. Zakas)
./node_modules/eslint/conf/eslint.json:        "consistent-return": "off",
./node_modules/fsevents/node_modules/extend/.eslintrc:		"consistent-return": [2],
./node_modules/fsevents/node_modules/qs/.eslintrc:		"consistent-return": [1],
```

**Comment by hkdobrev** [comment]
OK, removing `node_modules` and running `npm install` fixed the issue. Previously I had these versions:

```
xo@0.12.1
eslint@1.10.3
```

and now I have these:

```
xo@0.13.0
eslint@2.5.3
```
